pkg/tcpip/header/ndp_options.go

```diff
@@ -541,6 +571,10 @@ func (o NDPPrefixInformation) ValidLifetime() time.Duration {
 	return time.Second * time.Duration(binary.BigEndian.Uint32(o[ndpPrefixInformationValidLifetimeOffset:]))
 }
 
+func (o NDPPrefixInformation) SetValidLifetime(d time.Duration) {
```

Comment:
Looks like these lifetime setters are only using the seconds of the duration passed in. Can we use a `uint32` to represent time in units of seconds so the setters don't need to worry about stripping off any duration less than a second?

Here and other setters in this change